### ISSUE TITLE
Voice but not checkpoint for italian?

### ISSUE THREAD

**Issue by kirys** [rhasspy/piper#627]
Hi all
I'm using pipertts for a small experiment at home, I'm Italian so I'm using the Italian voice "Paola".
I was thinking of training another voice, so I started searching tutorials, but the ones I found starts from a checkpoint.
But in the repository i don't see an Italian checkpoint
https://huggingface.co/datasets/rhasspy/piper-checkpoints/tree/main
What should I do?
Thank You
K.

**Comment by Milkom121** [comment]
same problem here

**Comment by GianlucaApollaro** [comment]
Hi. I have the same problem. I would like to train an Italian voice, but because I don't have a check point everything sound like gibberish.

**Comment by kirys** [comment]
I tried using the Spanish checkpoint, but the results were weird (and wasted 10€ of colab credit ^_^).
I wonder how much GPU time would take to make a Italian checkpoint

**Comment by GianlucaApollaro** [comment]
Hi. I could finally make an Italian check point based on an English one and it worked quite well. Sometimes it gets weird, but I think that I could use a longer dataset to train even better. I'm not using colab, but I have my own GPU.

**Comment by kirys** [comment]
Mind sharing some info like 
How many minutes/hours of voice did it take?

**Comment by GianlucaApollaro** [comment]
Hi, for the moment I used around 40 minutes, and it doesn't sound that bad. I am planning to get to 2 hours of material in the near future.

**Comment by TeoColuccio** [comment]
> Hi. I could finally make an Italian check point based on an English one and it worked quite well. Sometimes it gets weird, but I think that I could use a longer dataset to train even better. I'm not using colab, but I have my own GPU.

Hi, did you used the Lessac checkpoints and fine-tuning Italian?

**Comment by GianlucaApollaro** [comment]
yes, it worked quite well, once I found the right amount of material.

**Comment by aventrax** [comment]
Hi guys, I would like to have another Italian Voice too, could you share your results? The project has only 2 voices but with strong accent that I don't like much.

**Comment by kirys** [comment]
I'm working on it, I'm about at 1700 epoch I'll publish on hugging face as soon as it reach 2000

**Comment by GianlucaApollaro** [comment]
Hello, I'm using a voice of a friend who's a voice actor, but unfortunately I don't have permission to release it. I'll try to find some free voices that I can use.

**Comment by kirys** [comment]
I've almost finished training the first voice based on the dataset I published here
https://huggingface.co/datasets/kirys79/piper_italiano
It has nice quality but an accent of south italy (Siciliano), but should be much better for fine tuning than any other language. 
Tomorrow I'll load it on this repo
https://huggingface.co/kirys79/piper_italiano

**Comment by gulp79** [comment]
Thanks for your work, i've tried these voices but atm i prefer Paola.
Btw is really nice to have other italian voices and some people working on this project.
